processor/tailsamplingprocessor/factory.go

```diff
@@ -31,6 +33,9 @@ const (
 
 // NewFactory returns a new factory for the Tail Sampling processor.
 func NewFactory() component.ProcessorFactory {
+	// TODO: this is hardcoding the metrics level and skips error handling
+	_ = view.Register(SamplingProcessorMetricViews(configtelemetry.LevelNormal)...)
```

Comment:
To avoid registering multiple times, maybe better to use a "once" variable to protect this.

Reply:
Oh, good idea. I used `sync.Once` and just updated the branch